tests/PolygonClipper.Tests/GenericTestCases.cs

```diff
@@ -88,6 +88,13 @@ private static Polygon ConvertToPolygon(IGeometryObject geometry)
                 }
 
                 polygon.Push(contour);
+
+                bool isLinear = ring.IsLinearRing();
+
+                if (!ring.IsClosed())
```

Comment:
This fixes Issue76. BUT collapsed_edges_removed failed now. Throwing that on the rust implementation also produces an error.

SO i am unsure what to do..

Reply:
Perhaps we should make 2 tests. 

One that has all the tests from the js version. Other one with the rust ones. 

Your goal is to get all green or just js or rust?

Reply:
I think we should stick with the Rust tests as that port is an evolution of the JS one and appears to be more correct.